user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrading_Downgrading_Webapps.md

```diff
@@ -0,0 +1,24 @@
+---
+uid: Upgrading_Downgrading_Webapps
+---
+
+# Upgrading the DataMiner web apps
+
+From DataMiner 10.3.0/10.3.3 onwards, DataMiner web upgrades are available separately from the general DataMiner upgrades. These updates include only the web API and the web apps, and typically take about 1 minute to install. This way you can get access to the latest features and enhancements of the web apps without having to do a full DataMiner upgrade. All other DataMiner processes remain untouched.
```

Comment:
Would not say how long it takes to install as this can vary going forward. It's also prolonged a bit since backups are now also taken during the web upgrade. Instead we can just say it doesn't take a lot of time.

One of the biggest advantages of web upgrades is that it doesn't do a DataMiner restart, so there is no monitoring or orchestrating downtime. I believe we should also explicitly mention it here.

Reply:
Thanks for the feedback! Changes made in commit [7bf9765](https://github.com/SkylineCommunications/dataminer-docs/pull/2093/commits/7bf9765eb3bbdc1e33967151e8220976b39ec508).